resources/views/footer-bar-desktop.blade.php

```diff
@@ -4,13 +4,15 @@
     'copyClass'    => '',
     'copyText'     => null,
     'socials'      => null,
+    'extra'        => null,
```

Comment:
this can use a better name. `extra` does not tell me it's a slot for the copyright part in the footer

Reply:
Updated @ItsANameToo